sql/catalyst/src/main/antlr3/org/apache/spark/sql/catalyst/parser/ExpressionParser.g

```diff
@@ -518,6 +527,10 @@ principalIdentifier
     | QuotedIdentifier
     ;
 
+looseNonReserved
+    : nonReserved | KW_FROM | KW_TO
+    ;
+
```

Comment:
We are allowed to use `From` and `To` in CreateTableUsing command's options. But we can't simply add them into `nonReserved` because by doing that we mess other existing rules. So we create a `looseIdentifier` and `looseNonReserved` here.